src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -189,9 +189,11 @@ def _getNumberOfTiles(self):
         # The size of the smallest tile, non-including the overlap, which will be
         # lost (and also indirectly represents the precision of the stage)
         reliable_fov = ((1 - self._overlap) * self._sfov[0], (1 - self._overlap) * self._sfov[1])
-
-        nx = math.ceil(abs(self._area_size[0] / reliable_fov[0]))
-        ny = math.ceil(abs(self._area_size[1] / reliable_fov[1]))
+        logging.debug("Would need tiles: nx= %s, ny= %s",
+                      abs(self._area_size[0] / reliable_fov[0]),
+                      abs(self._area_size[1] / reliable_fov[1]))
+        nx = math.floor(abs(self._area_size[0] / reliable_fov[0]))
+        ny = math.floor(abs(self._area_size[1] / reliable_fov[1]))
```

Comment:
No. This doesn't make sense. If we need more than C tiles, we need C+1 tiles. We cannot round down. BTW, this would break several test cases.

The only thing that we could do is to slightly reduce the area_size (eg, `* 0.999` or `- reliable_fov * 0.01`), to make sure that if precisely N tiles are requested, it's not round up due to floating point error.